adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -41,6 +41,102 @@ import scala.collection.JavaConversions._
 private[adam] class FastqRecordConverter extends Serializable with Logging {
 
   /**
+   * Parse 4 lines at a time
+   * @see parseReadPairInFastq
+   * *
+   */
+  private def parseReadInFastq(input: String,
+                               setFirstOfPair: Boolean = false,
+                               setSecondOfPair: Boolean = false,
+                               stringency: ValidationStringency = ValidationStringency.STRICT): (String, String, String) = {
+    val lines = input.split('\n')
+    require(lines.length == 4,
+      s"Input must have 4 lines (${lines.length.toString} found):\n${input}")
+
+    val readName = lines(0).drop(1)
+    if (readName.endsWith("/1") && setSecondOfPair)
```

Comment:
I've seen files in the wild that use ` 1` and ` 2` (with space) instead of `/1` and `/2`.  Could we add that here?
See e.g. [PairedEndFastqReader.java#L59](https://github.com/heuermh/dishevelled-bio/blob/master/read/src/main/java/org/dishevelled/bio/read/PairedEndFastqReader.java#L59)